lib/omnibus/builder.rb

```diff
@@ -795,12 +799,18 @@ def ship_license(name_or_url)
           "ZPL" => "https://gist.githubusercontent.com/remh/d60434c9fee49af69850/raw/5582f08b89995ee25bb0a556e32ca8a9de197f23/ZPL.license",
           "MIT" => "https://www.r-project.org/Licenses/MIT",
         }
-        url = (urls.key? name_or_url) ? urls[name_or_url] : name_or_url
-        raise "License #{url} is not starting with http" unless url.start_with? "http"
-        file_name = "#{dir_name}/" + url.split("/")[-1]
-        File.open(file_name, "wb") do |f|
-            log.info(log_key) { "Writing license file from #{url} to #{file_name}" }
-            f.write HTTParty.get(url).parsed_response
+        path = (known_licenses.key? source) ? known_licenses[source] : source
+        destination = "#{dir_name}/" + path.split("/")[-1]
+        if path.start_with? "http"
+          File.open(destination, "wb") do |f|
+            log.info(log_key) { "Downloading license file from #{path} to #{destination}" }
+            f.write HTTParty.get(path).parsed_response
+          end
+        else
+          Dir.chdir(software.project_dir) do
```

Comment:
✨

Reply:
review notes: https://www.rubydoc.info/github/opscode/omnibus/Omnibus%2FBuilder:software https://www.rubydoc.info/github/opscode/omnibus/Omnibus/Software#project_dir-instance_method

Reply:
👍 thanks for adding these